cpp/Platform.Random/RandomExtensions.h

```diff
@@ -1,11 +1,17 @@
 ﻿namespace Platform::Random
 {
-    class RandomExtensions
+    static std::uint64_t NextUInt64(const std::mt19937_64& random)
     {
-        public: static std::uint64_t NextUInt64(System::Random random) { return random.NextUInt64(Range.std::uint64_t); }
+        return NextUInt64(random, Range::UInt64);
+    }
```

Comment:
It is better to do:
```cpp
    static std::uint64_t NextUInt64(const std::mt19937_64& random)
    {
        static const std::uniform_int_distribution<std::uint64_t> generator(Range::UInt64.Minimum, Range::UInt64.Maximum);
        return generator(random);
    }
```

Reply:
Also remove const ref

Reply:
It does not give significant performance difference and makes more code.